sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -49,6 +50,8 @@ public async Task CreateAndGetAlertConfigurationWithWholeSeriesScope(bool useTok
             Assert.That(createdConfig.Description, Is.Empty);
             Assert.That(createdConfig.ConditionOperator, Is.Null);
             Assert.That(createdConfig.IdsOfHooksToAlert, Is.Not.Null.And.Empty);
+            // https://github.com/Azure/azure-sdk-for-net/issues/22433
+            //Assert.That(createdConfig.DimensionsToSplitAlert.Single(), Is.EqualTo(TempDataFeedDimensionNameA));
```

Comment:
Service bug (https://github.com/Azure/azure-sdk-for-net/issues/22433). Already reported.